website/blog/2019-05-13-the-new-docker-image-verdaccio-4.md

```diff
@@ -0,0 +1,60 @@
+---
+author: Juan Picado
+authorURL: https://twitter.com/jotadeveloper
+authorFBID: 1122901551
+title: The new docker image for Verdaccio 4
+---
+
+Docker has been for Verdaccio a key part of his success, at the time of this writing we have more than 4 millions and growing rapidly. The image provides an easy installation and integration with other tools, as Kubernetes, or composing them with any other Docker image. 
```

Comment:
```suggestion
Docker has been a key part of success for Verdaccio. At the time of this writing we have more than 4 millions image pulls per month and this number is growing rapidly. The image provides an easy way to use Verdaccio in combination with tools like Kubernetes, Docker Compose or any other container orchestration system, simplifying deployment and integration with an existing infrastructure.
```

Reply:
I like it, but there are 2 minor issues, this is the final test, could you update?
```
Docker has been a key part of success for Verdaccio. At the time of this writing, we have more than 4 million image pulls and this number is growing rapidly. The image provides an easy way to use Verdaccio in combination with tools like Kubernetes, Docker Compose or any other container orchestration system, simplifying deployment and integration with existing infrastructure.
```

Reply:
Sure